### DISCUSSION TITLE
Nicheformer example does not work

### DISCUSSION THREAD

**Discussion by xim2016** [community/community#120897] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Dear Nicheformer team members,

We are a cancer research group in University of Pittsburgh working on single cell omic data. We are very exceited to see your release of the paper Nicheformer which pretrained a model on hundreds of millions of single cell data. This will revolutionally pave the way for single cell down stream analysis.  We are eagle to try your model. Unfortunatally, the example notebook in docs/notebooks folder does not work properly. It poped up an error message: "module 'nicheformer' has no attribute 'pp'" . Could you please kindly give us some instruction about how to use your model. Greatly appreciated!

Thank you!
Xiaojun

**Comment by ebndev** [comment]
Hi @xim2016,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic or hoping to reach Nicheformer, you might want to try asking for help somewhere that focuses on this project. GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
